bundles/org.openhab.binding.openweathermap/README.md

```diff
@@ -445,16 +443,15 @@ Number:Temperature localDay6EveningApparent "Evening apparent Temperature [%.1f
 Number:Temperature localDay6NightApparent "Night apparent Temperature [%.1f %unit%]"  <temperature> { channel="openweathermap:onecall:api:local:forecastDay6#apparent-night" }
 Number:Pressure localDay6Pressure  "Barometric pressure [%.1f %unit%]"  <pressure> { channel="openweathermap:onecall:api:local:forecastDay6#pressure" }
 Number:Dimensionless localDay6Humidity  "Atmospheric humidity [%d %unit%]"  <humidity> { channel="openweathermap:onecall:api:local:forecastDay6#humidity" }
-Number:Temperature localDay6Dewpoint "Dew point [%.1f %unit%]"  <Temperature> { channea="openweathermap:onecall:api:local:forecastDay6#dew-point" }
+Number:Temperature localDay6Dewpoint "Dew point [%.1f %unit%]"  <Temperature> { channel="openweathermap:onecall:api:local:forecastDay6#dew-point" }
 Number:Speed localDay6WindSpeed  "Wind speed [%.1f km/h]"   <wind> { channel="openweathermap:onecall:api:local:forecastDay6#wind-speed" }
 Number:Angle localDay6WindDirection "Wind direction [%d %unit%]" <wind> { channel="openweathermap:onecall:api:local:forecastDay6#wind-direction" }
 Number:Speed localDay6GustSpeed "Gust speed [%.1f km/h]" <wind> { channel="openweathermap:onecall:api:local:forecastDay6#gust-speed" }
-Number:Dimensionless localDay6PrecipProbability  "Precipitation probability [%.1f]"    { channel="openweathermap:onecall:api:local:forecastDay6#probability" }
```

Comment:
There is a small typo: Should be `precip-probability`.

Reply:
done